Art/mrcybersecurett/index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <link rel="stylesheet" href="anime2.css">
```

Comment:
# Unused css file - anime.css
Your contribution:
- adds a file `anime.css` which is not used and `anime2.css` which is used.
   - if you are not using a file, please remove it
   - good convention and review here - would be to use anime.css file ( with the anime2.css content )
   then ensure the HTML links the correct file once your code reviewed by yourself.